portfolio/src/main/java/com/google/sps/servlets/MapServlet.java

```diff
@@ -40,32 +40,52 @@
   private static final String DESCRIPTION = "description";
   private static final String TABLE_NAME = "Pins";
   private static final String TIME_STAMP = "timestamp";
+  private static final String ID = "id";
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    UserService userService = UserServiceFactory.getUserService();
+    Query query = new Query(TABLE_NAME);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    List<UserComment> userCommentsList = new ArrayList<UserComment>();
```

Comment:
Nit: using "userComments" as variable name should be good enough.

Reply:
Changed